oxenss/server/omq.cpp

```diff
@@ -361,7 +321,7 @@ void OMQ::handle_client_request(std::string_view method, oxenmq::Message& messag
 
     try {
         std::string_view params = message.data.size() == full_size ? message.data.back() : ""sv;
-        it->second(
+        it->second.omq(
```

Comment:
will this properly throw if `it == rpc::RequestHandler::client_rpc_endpoints.end()` ?

Reply:
It shouldn't be possible because the code calling this is responsible for only passing in a valid method; there's a C-assert earlier in the function to check it in debug builds, but the check shouldn't be needed.